pkg/controllers/awsloadbalancercontroller/controller.go

```diff
@@ -33,12 +35,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/predicate"
-
-	albo "github.com/openshift/aws-load-balancer-operator/api/v1alpha1"
-	"github.com/openshift/aws-load-balancer-operator/pkg/aws"
 )
 
-const allowedResourceName = "cluster"
+const (
+	controllerName               = "cluster"
```

Comment:
@arjunrn I know the controller name was decided to be `cluster` in the EP, but is there any reason why we decided on it being called cluster(other than it denoting a single instance across the cluster)?  Because naming other related resources gets kind of weird (sa to be called `cluster-sa`).

Reply:
Not necessarily. Since names of different types can be the same. You can give them the same name. So the name of the service account would be `albo-cluster`. It can be some common prefix(albo) followed by the parent resource name i.e. `cluster.